### DISCUSSION TITLE
Having Transverter-Offset for API

### DISCUSSION THREAD

**Discussion by dg9vh** [magicbug/Cloudlog#1122] [Ideas / Feature Requests]
I am using a FT-817 operating at 70cm for uplink to QO-100. Here I also use a transverter putting my signal to 2400 MHz. To use CloudlogCat I need a feature to put a transverter-offset on the frequency delivered by Omnirig to CloudlogCat. I think having this configurable maybe on the API-KEY used with this setup would help - I could also imagine some other positions (maybe in the log-windows itself that supersedes the wrong frequency by the frequency with 1968 MHz offset.

How are you thinking about this? How could this problem be solved (to avoid manual frequency-changes in the log-windows)?

73 de Kim
DG9VH

**Comment by magicbug** [comment]
If memory serves me CloudlogCAT has offset for transverters built in, should just be a matter of adding them under "Settings" i think

**Comment by magicbug** [comment]
Just checked this and indeed its already there see https://github.com/magicbug/CloudlogCAT/releases

**Comment by dg9vh** [comment]
Really? I have the ic7300 test version installed because 2nd trx is IC
7300... Must have a look at it later when back home.

Kim

Peter Goodhall ***@***.***> schrieb am Mo., 2. Aug. 2021,
15:25:

**Comment by dg9vh** [comment]
On the IC7300-Version that is your last release, I can't find the option on
the GUI... where should I look for it?

Am Mo., 2. Aug. 2021 um 16:10 Uhr schrieb Kim Huebel ***@***.***>:

> Really? I have the ic7300 test version installed because 2nd trx is IC
> 7300... Must have a look at it later when back home.
>
> Kim
>
> Peter Goodhall ***@***.***> schrieb am Mo., 2. Aug. 2021,
> 15:25:
>
>> If memory serves me CloudlogCAT has offset for transverters built in
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/magicbug/Cloudlog/discussions/1122#discussioncomment-1120232>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/ADKN5OTZI7HJOSFBENHETJTT22MC3ANCNFSM5BMRXPDA>
>> .
>>
>

-- 
Amateurfunk ist das, was uns verbindet! Bleiben wir in Kontakt!
Ortsverband Q03, Neunkirchen
